### PR TITLE
Jnci content reorg2 062019

### PR DESCRIPTION
**Pull request by jamesneimanconsulting**
@pfista This version supersedes last week's PR and is meant to address the comments made during our discussion on 6/24.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,66 +1,31 @@
...
-
-`urlForAuthentication()` takes in an `options` object, which must have a `redirectURI` property defined. Other supported, but optional, properties are:
-
-* `loginHint` - The user's email address, if known.
-* `state` - An arbitrary string that will be returned back as a query param in your `redirectURI`.
-* `scopes` - An array of which scopes you'd like to auth with. The Nylas API provides granular authentication scopes that empower users with control over what level of access your application has to their data. See supported [Authentication Scopes](https://docs.nylas.com/docs/authentication-scopes) for a full list of scopes and details behind the scopes. If omitted, defaults to all scopes.
-
-### Step 1: Redirect the user to Nylas
+Every resource (i.e. messages, events, contacts, etc.) is accessed via an instance of ```Nylas```. Before making any requests, call ```config``` and initialize the Nylas instance with your APP ID and APP Secret. Then, call ```with``` and pass it your access token. The access token allows Nylas to make requests for a given email account's resources.
```
Comment (pfista) [inline README.md]: Unfortunately App ID and App Secret are not consistent with the naming in our API docs, which uses `client_id` and `client_secret`. We may want to make a quick note here to note the distinction (and that they are equivalent). Eventually we'll release a new SDK version that uses `client_id` instead.

Comment (jamesneimanconsulting) [reply README.md]: @pfista This is resolved, as well as the other comments.

**Comment by jamesneimanconsulting** [comment]
@pfista I've merged nylas master into this branch. Still seeing "This branch cannot be rebased due to conflicts. Only those with write access to this repository can merge pull requests." Perhaps this will help you resolve on your side?

**Comment by Gerdie** [comment]
@BenLloydPearson is working on more README adjustments

**Pull request closed without merging**